edx_proctoring/views.py

```diff
@@ -469,18 +469,29 @@ def get(self, request, course_id):
             allowed_enrollments_users
         ).values('user_id', 'status', 'modified')
 
+        # get all of the last verified onboarding attempts of these users
+        last_verified_attempt_dict = get_last_verified_onboarding_attempts_per_user(
+            allowed_enrollments_users,
+            onboarding_exam.backend,
+        )
+
         # select a verified, or most recent, exam attempt per user
         onboarding_attempts_per_user = self._get_relevant_attempt_per_user(onboarding_attempts)
 
         onboarding_data = []
         for user in allowed_enrollments_users:
             user_attempt = onboarding_attempts_per_user.get(user.id, {})
+            other_verified_attempt = last_verified_attempt_dict.get(user.id)
 
-            data = {}
-            data['username'] = user.username
-            data['status'] = (InstructorDashboardOnboardingAttemptStatus
-                              .get_onboarding_status_from_attempt_status(user_attempt.get('status')))
-            data['modified'] = user_attempt.get('modified')
+            data = {'username': user.username}
+
+            if not user_attempt and other_verified_attempt:
```

Comment:
I think this differs from the logic of the user view where we have `verified in another course` taking precedence over non-verified states in the current course.

My gut is that we want to replicate that here as well.

Reply:
I understand that logic differently than what you said here. See https://github.com/edx/edx-proctoring/blob/master/edx_proctoring/views.py#L374
That logic says to me, only if the learner have no attempt in the current course, we would check the verified attempt in other courses. I think that makes more sense because we want to show learner's status within this course.
Do you understand this differently?

Reply:
ahh you're right, this is the same in both places